tests/unit/python/foglamp/services/core/api/test_common_ping.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test rest server api for python/foglamp/services/core/api/common.py """
+
+from aiohttp import web
+import json
+import pytest
+
+from foglamp.services.core import routes
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("api", "common")
+async def test_ping(test_server, test_client, loop):
+    app = web.Application()
+    # fill route table
+    routes.setup(app)
+
+    server = await test_server(app)
+    server.start_server(loop=loop)
```

Comment:
@ashwinscale atleast server should be tested once, at ping. Later tests should use this tested server only (no need to use test_server fixture)

Reply:
Why not use both TestServer and TestClient fixtures for all tests?

Reply:
server shall be started behind the scene even with test_client fixture itself. Here the purpose is to test that with default start and config based https (To be done in next PR, as I pointed in PR description) 

Hence test_server fixture is explicitly used. And we don't need to test that server start again  with diff config, as that unit will be tested here itself.